### ISSUE TITLE
v0.6.0 Release Plan and Date

### ISSUE THREAD

**Issue by jdantonio** [ruby-concurrency/concurrent-ruby#82]
TL;DR I think we should remove `ActorContext` from the master branch and release 0.6.0 ASAP, possibly as soon as 5/25.

A quick look at the [changelog](https://github.com/jdantonio/concurrent-ruby/wiki/Changelog) shows that we have implemented a _ton_ of great stuff since the 0.5.0 release. The only thing we're waiting on that was planned for 0.6.0 is a new actor implementation. Since I've changed direction on that (again, more on this later) I think we should remove that from the roadmap for 0.6.0. We've marked the current `Actor` implementation as deprecated but haven't removed it. We lose nothing if we release 0.6.0 without a new actor implementation, but we deny our community access to many great updates if we continue to hold back.

A further discussion of actors can be found in [this thread](https://github.com/jdantonio/concurrent-ruby/issues/81).

Thoughts?
